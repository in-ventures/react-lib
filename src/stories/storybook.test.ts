/*
 * File: storybook.test.ts
 * Project: @inventures/react-lib
 * File Created: Thursday, 3rd September 2020 3:01:00 pm
 * Author: Gabriel Ulloa (gabriel@inventures.cl)
 * -----
 * Last Modified: Thursday, 3rd September 2020 3:01:15 pm
 * Modified By: Gabriel Ulloa (gabriel@inventures.cl)
 * -----
 * Copyright 2019 - 2020 Incrementa Ventures SpA. ALL RIGHTS RESERVED
 * Terms and conditions defined in license.txt
 * -----
 * Inventures - www.inventures.cl
 */

import initStoryshots from '@storybook/addon-storyshots';
initStoryshots();
