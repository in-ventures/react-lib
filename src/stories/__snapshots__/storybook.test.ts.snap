// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Card Base element 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-root-2 MuiPaper-elevation1 MuiPaper-rounded"
>
  <button
    className="MuiButtonBase-root MuiCardActionArea-root"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <div
      className="MuiCardMedia-root makeStyles-media-5"
      onClick={[Function]}
      style={
        Object {
          "backgroundImage": "url(\\"https://www.laboratoriochile.cl/wp-content//uploads/2015/11/Clonazepam_05MG_30C_BE_HD.jpg\\")",
        }
      }
    />
    <div
      className="MuiChip-root makeStyles-disabled-12 MuiChip-sizeSmall"
      onKeyDown={[Function]}
      onKeyUp={[Function]}
    >
      <span
        className="MuiChip-label MuiChip-labelSmall"
      />
    </div>
    <div
      className="MuiCardContent-root makeStyles-content-4"
      onClick={[Function]}
    >
      <h5
        className="MuiTypography-root makeStyles-title-3 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
      >
        Clonazepam · 2 mg
      </h5>
      <div
        className="MuiBox-root MuiBox-root-13 makeStyles-textBox-7"
      >
        subtitle
      </div>
      <div
        className="MuiBox-root MuiBox-root-14 makeStyles-textBox-7"
      >
        Clonazepam · 2 mg · 30 comprimidos
      </div>
      <div
        className="MuiBox-root MuiBox-root-15 makeStyles-textBox-7"
      />
      <h6
        className="MuiTypography-root makeStyles-price-6 MuiTypography-body1 MuiTypography-colorTextPrimary"
      >
        CLP 5,025
      </h6>
    </div>
    <span
      className="MuiCardActionArea-focusHighlight"
    />
  </button>
</div>
`;

exports[`Storyshots Card Editable Card 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-root-16 MuiPaper-elevation1 MuiPaper-rounded"
>
  <button
    className="MuiButtonBase-root MuiCardActionArea-root"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <div
      className="MuiCardMedia-root makeStyles-media-19"
      onClick={[Function]}
      style={
        Object {
          "backgroundImage": "url(\\"https://www.cruzverde.cl/dw/image/v2/BDPM_PRD/on/demandware.static/-/Sites-masterCatalog_Chile/default/dw0ebcdb64/images/large/296432-okrafit-120-capsulas.jpg?sw=1000&sh=1000\\")",
        }
      }
    />
    <div
      className="MuiChip-root MuiChip-colorPrimary MuiChip-sizeSmall"
      onKeyDown={[Function]}
      onKeyUp={[Function]}
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiChip-icon MuiChip-iconSmall MuiChip-iconColorPrimary"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zM6 20V4h7v5h5v11H6z"
        />
      </svg>
      <span
        className="MuiChip-label MuiChip-labelSmall"
      >
        Receta retenida
      </span>
    </div>
    <div
      className="MuiCardContent-root makeStyles-content-18"
      onClick={[Function]}
    >
      <h5
        className="MuiTypography-root makeStyles-title-17 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
      >
        Glafornil
      </h5>
      <div
        className="MuiBox-root MuiBox-root-27 makeStyles-textBox-21"
      >
        Metformina Clorhidrato Clorhidrato
      </div>
      <div
        className="MuiBox-root MuiBox-root-28 makeStyles-textBox-21"
      >
        500 mg
      </div>
      <div
        className="MuiBox-root MuiBox-root-29 makeStyles-textBox-21"
      >
        30 comprimidos recubiertos
      </div>
      <h6
        className="MuiTypography-root makeStyles-price-20 MuiTypography-body1 MuiTypography-colorTextPrimary"
      >
        CLP 15,990
      </h6>
    </div>
    <span
      className="MuiCardActionArea-focusHighlight"
    />
  </button>
</div>
`;

exports[`Storyshots Card Product Carousel 1`] = `
<div
  className="MuiBox-root MuiBox-root-41"
>
  <div
    className="MuiGrid-root makeStyles-gridContainer-38 MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
  >
    <h5
      className="MuiTypography-root makeStyles-title-31 MuiTypography-body1 MuiTypography-colorTextPrimary"
    >
      Más vendidos
    </h5>
    <button
      aria-label="view product categories"
      className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-39 MuiIconButton-colorPrimary"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M9.29 6.71c-.39.39-.39 1.02 0 1.41L13.17 12l-3.88 3.88c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0l4.59-4.59c.39-.39.39-1.02 0-1.41L10.7 6.7c-.38-.38-1.02-.38-1.41.01z"
          />
        </svg>
      </span>
    </button>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
      >
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-30 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <button
              className="MuiButtonBase-root MuiCardActionArea-root"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <div
                className="MuiCardMedia-root makeStyles-media-33"
                onClick={[Function]}
                style={
                  Object {
                    "backgroundImage": "url(\\"https://www.cruzverde.cl/dw/image/v2/BDPM_PRD/on/demandware.static/-/Sites-masterCatalog_Chile/default/dw0ebcdb64/images/large/296432-okrafit-120-capsulas.jpg?sw=1000&sh=1000\\")",
                  }
                }
              />
              <div
                className="MuiChip-root MuiChip-colorPrimary MuiChip-sizeSmall"
                onKeyDown={[Function]}
                onKeyUp={[Function]}
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiChip-icon MuiChip-iconSmall MuiChip-iconColorPrimary"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zM6 20V4h7v5h5v11H6z"
                  />
                </svg>
                <span
                  className="MuiChip-label MuiChip-labelSmall"
                >
                  Receta retenida
                </span>
              </div>
              <div
                className="MuiCardContent-root makeStyles-content-32"
                onClick={[Function]}
              >
                <h5
                  className="MuiTypography-root makeStyles-title-31 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
                >
                  Glafornil
                </h5>
                <div
                  className="MuiBox-root MuiBox-root-42 makeStyles-textBox-35"
                >
                  Metformina Clorhidrato Clorhidrato
                </div>
                <div
                  className="MuiBox-root MuiBox-root-43 makeStyles-textBox-35"
                >
                  500 mg
                </div>
                <div
                  className="MuiBox-root MuiBox-root-44 makeStyles-textBox-35"
                >
                  30 comprimidos recubiertos
                </div>
                <h6
                  className="MuiTypography-root makeStyles-price-34 MuiTypography-body1 MuiTypography-colorTextPrimary"
                >
                  CLP 15,990
                </h6>
              </div>
              <span
                className="MuiCardActionArea-focusHighlight"
              />
            </button>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-30 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <button
              className="MuiButtonBase-root MuiCardActionArea-root"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <div
                className="MuiCardMedia-root makeStyles-media-33"
                onClick={[Function]}
                style={
                  Object {
                    "backgroundImage": "url(\\"https://www.cruzverde.cl/dw/image/v2/BDPM_PRD/on/demandware.static/-/Sites-masterCatalog_Chile/default/dw0ebcdb64/images/large/296432-okrafit-120-capsulas.jpg?sw=1000&sh=1000\\")",
                  }
                }
              />
              <div
                className="MuiChip-root makeStyles-disabled-40 MuiChip-sizeSmall"
                onKeyDown={[Function]}
                onKeyUp={[Function]}
              >
                <span
                  className="MuiChip-label MuiChip-labelSmall"
                />
              </div>
              <div
                className="MuiCardContent-root makeStyles-content-32"
                onClick={[Function]}
              >
                <h5
                  className="MuiTypography-root makeStyles-title-31 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
                >
                  Glafornil
                </h5>
                <div
                  className="MuiBox-root MuiBox-root-45 makeStyles-textBox-35"
                >
                  Metformina Clorhidrato Clorhidrato
                </div>
                <div
                  className="MuiBox-root MuiBox-root-46 makeStyles-textBox-35"
                >
                  500 mg
                </div>
                <div
                  className="MuiBox-root MuiBox-root-47 makeStyles-textBox-35"
                >
                  30 comprimidos recubiertos
                </div>
                <h6
                  className="MuiTypography-root makeStyles-price-34 MuiTypography-body1 MuiTypography-colorTextPrimary"
                >
                  CLP 15,990
                </h6>
              </div>
              <span
                className="MuiCardActionArea-focusHighlight"
              />
            </button>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item"
        >
          <div
            className="MuiPaper-root MuiCard-root makeStyles-root-30 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <button
              className="MuiButtonBase-root MuiCardActionArea-root"
              disabled={false}
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <div
                className="MuiCardMedia-root makeStyles-media-33"
                onClick={[Function]}
                style={
                  Object {
                    "backgroundImage": "url(\\"https://www.cruzverde.cl/dw/image/v2/BDPM_PRD/on/demandware.static/-/Sites-masterCatalog_Chile/default/dw0ebcdb64/images/large/296432-okrafit-120-capsulas.jpg?sw=1000&sh=1000\\")",
                  }
                }
              />
              <div
                className="MuiChip-root MuiChip-colorPrimary MuiChip-sizeSmall"
                onKeyDown={[Function]}
                onKeyUp={[Function]}
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiChip-icon MuiChip-iconSmall MuiChip-iconColorPrimary"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M14 2H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8l-6-6zM6 20V4h7v5h5v11H6z"
                  />
                </svg>
                <span
                  className="MuiChip-label MuiChip-labelSmall"
                >
                  Receta retenida
                </span>
              </div>
              <div
                className="MuiCardContent-root makeStyles-content-32"
                onClick={[Function]}
              >
                <h5
                  className="MuiTypography-root makeStyles-title-31 MuiTypography-subtitle1 MuiTypography-colorTextPrimary"
                >
                  Glafornil
                </h5>
                <div
                  className="MuiBox-root MuiBox-root-48 makeStyles-textBox-35"
                >
                  Metformina Clorhidrato Clorhidrato
                </div>
                <div
                  className="MuiBox-root MuiBox-root-49 makeStyles-textBox-35"
                >
                  500 mg
                </div>
                <div
                  className="MuiBox-root MuiBox-root-50 makeStyles-textBox-35"
                >
                  30 comprimidos recubiertos
                </div>
                <h6
                  className="MuiTypography-root makeStyles-price-34 MuiTypography-body1 MuiTypography-colorTextPrimary"
                >
                  CLP 15,990
                </h6>
              </div>
              <span
                className="MuiCardActionArea-focusHighlight"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Cart Editable Cart Product 1`] = `
<div
  className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
>
  <button
    className="MuiButtonBase-root MuiCardActionArea-root"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <div
      className="makeStyles-root-51"
    >
      <div
        className="MuiCardMedia-root makeStyles-cover-55"
        style={
          Object {
            "backgroundImage": "url(\\"https://www.laboratoriochile.cl/wp-content//uploads/2015/11/Clonazepam_05MG_30C_BE_HD.jpg\\")",
          }
        }
        title=""
      />
      <div
        className="makeStyles-details-52"
      >
        <div
          className="MuiCardContent-root makeStyles-content-54"
        >
          <div
            className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom"
          >
            <div
              className="MuiBox-root MuiBox-root-60"
            >
              Nombre comercial
            </div>
          </div>
          <div
            className="MuiTypography-root makeStyles-smalltext-59 MuiTypography-body1 MuiTypography-gutterBottom"
          >
            <div
              className="MuiBox-root MuiBox-root-61"
            >
              100 mcg
            </div>
            <div
              className="MuiBox-root MuiBox-root-62"
            >
              50 comprimidos
            </div>
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-underline-58 MuiTypography-colorPrimary"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
            >
              Volverás a necesitar 05/08/20
            </a>
          </div>
          <div
            className="MuiTypography-root makeStyles-smalltext-59 MuiTypography-body1"
          >
            <div
              className="MuiBox-root MuiBox-root-63"
            >
              Valor unitario: $ 
              10
            </div>
          </div>
          <div
            className="MuiTypography-root MuiTypography-body1"
          >
            <div
              className="MuiBox-root MuiBox-root-64"
            >
              Total: $ 
              10
            </div>
          </div>
        </div>
      </div>
    </div>
    <span
      className="MuiCardActionArea-focusHighlight"
    />
  </button>
  <div
    className="MuiCardActions-root makeStyles-actions-53 MuiCardActions-spacing"
  >
    <button
      aria-label="delete"
      className="MuiButtonBase-root MuiIconButton-root makeStyles-trashicon-57 MuiIconButton-sizeSmall"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
          />
        </svg>
      </span>
    </button>
    <button
      aria-label="add"
      className="MuiButtonBase-root MuiIconButton-root makeStyles-sumicon-56"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root MuiIcon-colorPrimary MuiIcon-fontSizeLarge"
        >
          add_circle
        </span>
      </span>
    </button>
    <div
      className="MuiTypography-root MuiTypography-subtitle1"
    >
      <div
        className="MuiBox-root MuiBox-root-65"
      >
        1
      </div>
    </div>
    <button
      aria-label="sub"
      className="MuiButtonBase-root MuiIconButton-root makeStyles-sumicon-56"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <span
          aria-hidden={true}
          className="material-icons MuiIcon-root MuiIcon-colorPrimary MuiIcon-fontSizeLarge"
        >
          remove_circle
        </span>
      </span>
    </button>
  </div>
</div>
`;

exports[`Storyshots Input Base element 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
    />
  </div>
</div>
`;

exports[`Storyshots Input Input For Email 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
>
  <label
    className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
    data-shrink={false}
  >
    Mail* (800ms)
  </label>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
      value=""
    />
  </div>
</div>
`;

exports[`Storyshots Input Input For Phone 1`] = `
<div
  className="MuiFormControl-root makeStyles-formControl-1"
>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <div
      aria-haspopup="listbox"
      className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      role="button"
      tabIndex={0}
    >
      Chile
       (+
      56
      )
    </div>
    <input
      aria-hidden={true}
      className="MuiSelect-nativeInput"
      onAnimationStart={[Function]}
      onChange={[Function]}
      required={false}
      tabIndex={-1}
      value="Chile"
    />
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSelect-icon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 10l5 5 5-5z"
      />
    </svg>
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
    >
      Phone* (800ms)
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots Input Input For Rut 1`] = `
<div
  className="MuiFormControl-root MuiTextField-root"
>
  <label
    className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
    data-shrink={false}
  >
    RUT* (800ms)
  </label>
  <div
    className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      autoFocus={false}
      className="MuiInputBase-input MuiInput-input"
      disabled={false}
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="text"
      value=""
    />
  </div>
</div>
`;

exports[`Storyshots Input Input With Mask 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
    }
  }
>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
    >
      Maximo 4
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
    >
      fecha de nacimiento 
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
        value=""
      />
    </div>
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
    >
      Ingrese RUT (800ms)
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;

exports[`Storyshots SearchBar Base 1`] = `
<div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
    >
      Search (800ms)
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="Text"
        value=""
      />
    </div>
  </div>
  <ul />
</div>
`;
